webscrapbook/cli.py

```diff
@@ -282,6 +299,28 @@ def main():
         help="""generate more assistant files. (with --book)""")
     parser_config.add_argument('-e', '--edit', default=False, action='store_true',
         help="""edit the config file. (with --book or --user)""")
+    
+    # subcommand: scrapbook
+    parser_scrapbook = subparsers.add_parser('scrapbook', aliases=['scrap'],
```

Comment:
We may consider using 'book' and 'b' if we want a grouped sub-command.

Reply:
For sure I will change it to `wsb sort ...` since we can easily change it back if we need to.